api/client/gro_client_test.py

```diff
@@ -1,92 +1,373 @@
 try:
     # Python 3.3+
-    from unittest.mock import MagicMock
+    from unittest.mock import patch, MagicMock
 except ImportError:
     # Python 2.7
-    from mock import MagicMock
+    from mock import patch, MagicMock
+from unittest import TestCase
+from datetime import date
 
-import pytest
 from api.client.gro_client import GroClient
 
 MOCK_HOST = 'pytest.groclient.url'
 MOCK_TOKEN = 'pytest.groclient.token'
 
-client = GroClient(MOCK_HOST, MOCK_TOKEN)
+mock_entities = {
```

Comment:
import mock data from module

Reply:
:+1:

Reply:
Done. Left a todo comment about using conftest more to share mock data between different test files (that would let you mock out requests.get() generally and then you don't need specific mocking in test files at all), but for now I just put the mock data in its own module and imported it. Good first step